package.json

```
@@ -34,6 +35,7 @@
         "eslint-config-airbnb-typescript": "17.0.0",
         "eslint-plugin-editorconfig": "4.0.2",
         "eslint-plugin-react": "7.30.1",
+        "express": "^4.18.1",
```

```suggestion
        "express": "4.18.1",
```
предлагаю сразу фиксировать версии